sort.js

```diff
@@ -0,0 +1,99 @@
+
+
+//Insertion sort
+const insertion = (a) => {
```

Comment:
https://www.w3resource.com/javascript-exercises/searching-and-sorting-algorithm/searching-and-sorting-algorithm-exercise-4.php